### ISSUE TITLE
No such member when using package.preload

### ISSUE THREAD

**Issue by sosodev** [Benjamin-Dobell/IntelliJ-Luanalysis#70]
Environment
--------------
   name   |   version  
----------|----------
 IDEA version         | GO2020.3
 Luanalysis version | 1.2.2-IDEA203
 OS                        | Linux

What are the steps to reproduce this issue?
------------------------------------------------
1. Use package.preload to register a module

```Lua
package.preload['mymodule'] = function()
-- ...
end
```

What happens?
-------------
Luanalysis raises the error "No such member 'mymodule' found on type table"

What were you expecting to happen?
-----------------------------------------
No error :) 

Any other comments?
------------------------
This project is really awesome by the way. It's super cool to have something like this for Lua.

**Comment by adriweb** [comment]
You also do require it, right?
From your code sample: 

local mymodule = require("mymodule")

**Comment by sosodev** [comment]
This is in the context of an [application](https://github.com/Hyperspace-Logistics/heart) that preloads these modules before running user scripts would require these modules. They're not required internally, no.
